### PR TITLE
Checkbox

### PR DESCRIPTION
**Pull request by tylerlee**
adds a custom look to checkboxes
image

### CODE REVIEW

File: scripts/components/event-filters/list-item.es6 [hunk 1]
```diff
@@ -1,41 +1,22 @@
...
-          name={ICON_MAP[eventFilter.type]}
-          style={{color: '#' + eventFilter.hex}}
-        />
         <div className='osw-event-filters-list-item-name'>
-          <input
-            className='osw-event-filters-list-item-checkbox'
-            type='checkbox'
-            checked={eventFilter.active}
+          <Checkbox
+            cursors={{boolState: this.getCursor('eventFilter', 'active')}}
```
Comment (caseywebdev) [inline scripts/components/event-filters/list-item.es6]: `boolState` is a weird variable name. Just call it `checked`.

File: styles/ui/custom-checkbox.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+@import '../config';
+
+$sides: 14px;
```
Comment (caseywebdev) [inline styles/ui/custom-checkbox.scss]: This is also a questionable variable name. Does `$size` make more sense?

File: styles/ui/custom-checkbox.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    border-radius: 3px;
+    color: #fff;
+    display: inline-block;
+    height: $sides;
+    margin-right: 5px;
+    position: relative;
+    top: 3px;
+    width: $sides;
+
+    i {
```
Comment (caseywebdev) [inline styles/ui/custom-checkbox.scss]: `.osw-icon` (it's something like that) is probably a better selector to use.

File: styles/ui/custom-checkbox.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      left: ($sides - 12)/2;
+      position: absolute;
+      top: ($sides - 14)/2;
+    }
+  }
+
+  .osw-checkbox-unchecked {
```
Comment (caseywebdev) [inline styles/ui/custom-checkbox.scss]: Move these two into the `.orgsync-widget` section.

File: styles/ui/custom-checkbox.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    @include linear-gradient(#fff, $grey-lightest);
+    border: 1px solid #ccc;
+    box-shadow: inset 0 1px 1px #ccc;
+    color: #dedede;
+  }
+
+  .osw-checkbox-colored.osw-checkbox-unchecked {
+    background: none !important;
```
Comment (caseywebdev) [inline styles/ui/custom-checkbox.scss]: Can we avoid the `!important`?

**Comment by caseywebdev** [comment]
Some notes in the lines. Also, the check marks don't look centered. Also also, the "Portals" checkbox looking different than the others bothers me.

File: styles/ui/custom-checkbox.scss (hunk 5 above)
Comment (tylerlee) [inline styles/ui/custom-checkbox.scss]: I'm using inline styles to set the color so not really.

File: styles/ui/custom-checkbox.scss (hunk 2 above)
Comment (tylerlee) [inline styles/ui/custom-checkbox.scss]: picky picky picky

File: styles/ui/custom-checkbox.scss (hunk 4 above)
Comment (tylerlee) [inline styles/ui/custom-checkbox.scss]: they're in it

Comment (caseywebdev) [inline styles/ui/custom-checkbox.scss]: Just making sure you're paying attention

**Comment by tylerlee** [comment]
yeah i was getting your OK on the component, i'll change the HEADERS part now

File: styles/ui/custom-checkbox.scss (hunk 5 above)
Comment (caseywebdev) [inline styles/ui/custom-checkbox.scss]: You could *not* add the color in the style if it's not checked.

**Comment by tylerlee** [comment]
alright @caseywebdev . updated the checkbox component so it no longer has cursors in it. i think thats all there was left on this sucker. cool if i merge?

**Comment by caseywebdev** [comment]
:+1:

**Pull request merged**